src/SFA.DAS.LevyTransferMatching.Infrastructure/Services/PledgeService/Types/GetApplicationsResponse.cs

```diff
@@ -31,10 +31,11 @@ public class Application
             public DateTime CreatedOn { get; set; }
             public ApplicationStatus Status { get; set; }
             public bool IsNamePublic { get; set; }
-            public bool IsLocationMatch { get; set; }
-            public bool IsSectorMatch { get; set; }
-            public bool IsJobRoleMatch { get; set; }
-            public bool IsLevelMatch { get; set; }
+            public bool MatchSector { get; set; }
```

Comment:
See previous comment

Reply:
suggestions implemented with as little code changes as needed by this ticket